examples/composition-api/basic/pages/index.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="container">
+    <button @click="increment">
+      Increment
+    </button>
+    <p>Counter : {{ counter }}</p>
+    <button @click="decrement">
+      Decrement
+    </button>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import VueCompositionApi, { defineComponent, ref } from '@vue/composition-api'
+
+Vue.use(VueCompositionApi)
+
+export default defineComponent({
+  name: 'CompositionAPICounter',
```

Comment:
Is this required ?

Reply:
ln 17? `Vue.use(VueCompositionApi)` thats been removed in one of the other commits now.